vllm/model_executor/utils.py

```diff
@@ -11,3 +12,24 @@ def set_random_seed(seed: int) -> None:
     torch.manual_seed(seed)
     if torch.cuda.is_available():
         torch.cuda.manual_seed_all(seed)
+
+
+def set_weight_attrs(
+    weight: torch.Tensor,
+    weight_attrs: Optional[dict[str, Any]],
```

Comment:
Same for all other places.
```suggestion
    weight_attrs: Optional[Dict[str, Any]],
```